release v1.4/over-het-algoritmekader-tekst-en-opmaak (#193)
MinBZK/Algoritmekader: docs/overhetalgoritmekader/index.md

---
title: Over het Algoritmekader
summary: Het Algoritmekader is een open source website van het Ministerie van Binnenlandse Zaken, met alle regels voor overheden die algoritmes of AI gebruiken.
---
# Over het algoritmekader 

> Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin zoek je de regels en adviezen op per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een testversie. De definitieve versie verschijnt eind 2024.

Reviewer: Ook hier twijfel ik over het woord 'regels'. Klinkt toch een beetje alsof BZK regels oplegt, terwijl er geen nieuwe verplichtingen in het algoritmekader staan. Het is een overzicht van (de belangrijkste) bestaande vereisten volgens wet- en regelgeving of bestaand beleid, aangevuld met praktische suggesties over hoe je daar invulling aan kan geven.  Er is daar in de omgeving best wat onduidelijkheid over, dus ik vind het belangrijk dit ook duidelijk op de site te vermelden.

Author: ```suggestion
Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin vind je de wetten en regels, hulpmiddelen en adviezen per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een bètaversie. De definitieve versie verschijnt eind 2024.
```

## Verantwoord gebruik van algoritmes en AI
In het Algoritmekader vind je informatie over verantwoord gebruik van algoritmes en AI. Bijvoorbeeld:
- Wie is waarvoor verantwoordelijk?
- Hoe voorkom je discriminatie (bias) in een algoritme?
- Hoe verwerk je gegevens op een veilige manier?

> - Welke algoritmes controleer je op mensenrechtenschendingen?

Reviewer: ```suggestion
- Hoe voorkom je je mensenrechtenschendingen?
```

Reviewer: gaat niet per se om welke algoritmes dit zijn, maar meer hoe je mensenrechtenschending kan voorkomen, en hoe je er rekening mee kan houden in de ontwikkeling

Author: ```suggestion
- Hoe voorkom je mensenrechtenschendingen?
```

- Welke inkoopvoorwaarden spreek je af voor algoritmes van een externe partij?

De informatie is bedoeld voor medewerkers van de rijksoverheid, provincies, gemeentes en waterschappen.

## Informatie van de overheid
Het Algoritmekader is een website van het Ministerie van Binnenlandse Zaken en Koninkrijksrelaties. Het team Data, AI en Algoritmes stelt de informatie samen, op basis van:

> - alle wetten en regels, zoals de [AI-verordening](https://www.digitaleoverheid.nl/overzicht-van-alle-onderwerpen/nieuwe-technologieen-data-en-ethiek/artificiele-intelligentie-ai/ai-verordening/)

Reviewer: Misschien beter ter verwijzen naar de AI verordening zelf? https://eur-lex.europa.eu/legal-content/NL/TXT/HTML/?uri=OJ:L_202401689#d1e39-1-1

Author: Kan ook, maar dan moeten we deze link gebruiken: https://eur-lex.europa.eu/legal-content/NL/TXT/?uri=CELEX%3A32024R1689&qid=1723226664277

> - alle wetten en regels, zoals de [AI-verordening](https://www.digitaleoverheid.nl/overzicht-van-alle-onderwerpen/nieuwe-technologieen-data-en-ethiek/artificiele-intelligentie-ai/ai-verordening/)

Reviewer: ```suggestion
- alle wetten en regels, zoals de [AI-verordening](https://eur-lex.europa.eu/legal-content/NL/TXT/?uri=CELEX%3A32024R1689&qid=1723226664277) en de [Algemene wet bestuursrecht](https://wetten.overheid.nl/BWBR0005537/2022-11-01/1)
```

> - belangrijke instrumenten, zoals de [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes) (Impact Assessment voor Mensenrechten bij de inzet van Algoritmes)

Reviewer: ```suggestion
- belangrijke instrumenten, zoals de [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes) (Impact Assessment Mensenrechten en Algoritmes)
```

- adviezen en ervaringen van experts, zoals wetenschappers

> 

Author: Ik zou het zo opschrijven, akkoord? 
```suggestion
## Verschillende versies
Het Algoritmekader is de nieuwe naam van het [Implementatiekader 'Verantwoorde inzet van algoritmen'](https://www.rijksoverheid.nl/documenten/rapporten/2023/06/30/implementatiekader-verantwoorde-inzet-van-algoritmen). Dit rapport is de eerste versie. Door het verder ontwikkelen van dit rapport ontstond de website Algoritmekader.

Sindsdien verschijnt elke maand een nieuwe bètaversie van het Algoritmekader. Deze bètaversies zijn nog niet betrouwbaar. De informatie is onvolledig. Er kunnen fouten in staan.

Versie 2.0 is de definitieve, betrouwbare versie van het Algoritmekader. Deze versie verschijnt eind 2024.

## Iedereen mag meedenken
De informatie in het Algoritmekader komt open source tot stand. Dat betekent dat iedereen kan meekijken en zijn mening mag geven, of een voorstel mag doen:
```

## Iedereen mag meedenken

> Tot eind 2024 is het Algoritmekader een testversie. De informatie komt open source tot stand. Dat betekent dat iedereen zijn mening mag geven of een voorstel mag doen:

Reviewer: Misschien beter: in ontwikkeling, of in concept?

Author: ```suggestion
Tot eind 2024 is het Algoritmekader een bètaversie. De informatie is nog in ontwikkeling en komt open source tot stand. Dat betekent dat iedereen zijn mening mag geven of een voorstel mag doen:
```

> ## Iedereen mag meedenken
> Tot eind 2024 is het Algoritmekader een bètaversie. De informatie is nog in ontwikkeling en komt open source tot stand. Dat betekent dat iedereen zijn mening mag geven of een voorstel mag doen:

Reviewer: ```suggestion
```

Reviewer: komt te vervallen door bovenstaande wijziging

> - Geef je [vraag of wijziging](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/CONTRIBUTING/) door.

Reviewer: ```suggestion
- Geef je [vraag of wijziging](CONTRIBUTING.md) door.
```

- Sluit je aan bij een [werkgroep](https://algoritmes.pleio.nl/).

> - Doe mee aan een [online vergadering](https://algoritmes.pleio.nl/events) zoals onze sprintreview of demo.

Reviewer: ```suggestion
- Doe mee aan een [online vergadering](https://algoritmes.pleio.nl/events) zoals onze demo na een nieuwe release van het Algoritmekader.
```

Author: Is dat de enige vergadering waar mensen aan mee kunnen doen? Want dan is 'zoals' niet op zijn plaats.

Reviewer: Er zijn er meerdere (niet allemaal online). We houden alleen nooit een sprintreview, maar wel een demo!

## Nieuwsbrief
Via onze maandelijkse [Nieuwsbrief Algoritmes](https://algoritmeregister.email-provider.eu/memberforms/subscribe/standalone/form/?a=1pjwwoyxrs&l=vdfr1sbovb) blijf je op de hoogte over de ontwikkelingen van het Algoritmekader.

## Help ons deze pagina te verbeteren

> Deel je idee, suggestie of opmerking via [GitHub](https://github.com/MinBZK/Algoritmekader/edit/main/docs/overhetalgoritmekader/index.md) of mail ons via [algoritmes@minbzk.nl](mailto:algoritmes@minbzk.nl).

Reviewer: ```suggestion
Deel je idee, suggestie of opmerking via [GitHub](https://github.com/MinBZK/Algoritmekader) of mail ons via [algoritmes@minbzk.nl](mailto:algoritmes@minbzk.nl).
```

Author: Ik heb de specifieke edit-link gebruikt, onder het icoontje rechtsboven.

---
title: Over het Algoritmekader
summary: Het Algoritmekader is een open source website van het Ministerie van Binnenlandse Zaken, met alle regels voor overheden die algoritmes of AI gebruiken.
---
# Over het algoritmekader 
Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin vind je de wetten en regels, hulpmiddelen en adviezen per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een bètaversie. De definitieve versie verschijnt eind 2024.

## Verantwoord gebruik van algoritmes en AI

> In het Algoritmekader vind je informatie over verantwoord gebruik van algoritmes en AI. Bijvoorbeeld:
> - Wie is waarvoor verantwoordelijk?

Reviewer: ```suggestion
In het Algoritmekader vind je informatie over verantwoord gebruik van algoritmes en AI. Bijvoorbeeld:

- Wie is waarvoor verantwoordelijk?
```

- Hoe voorkom je discriminatie (bias) in een algoritme?
- Hoe verwerk je gegevens op een veilige manier?
- Hoe voorkom je mensenrechtenschendingen?
- Welke inkoopvoorwaarden spreek je af voor algoritmes van een externe partij?

De informatie is bedoeld voor medewerkers van de rijksoverheid, provincies, gemeentes en waterschappen.

## Informatie van de overheid

> Het Algoritmekader is een website van het Ministerie van Binnenlandse Zaken en Koninkrijksrelaties. Het team Data, AI en Algoritmes stelt de informatie samen, op basis van:
> - alle wetten en regels, zoals de [AI-verordening](https://eur-lex.europa.eu/legal-content/NL/TXT/?uri=CELEX%3A32024R1689&qid=1723226664277) en de [Algemene wet bestuursrecht](https://wetten.overheid.nl/BWBR0005537/2022-11-01/1)

Reviewer: ```suggestion
Het Algoritmekader is een website van het Ministerie van Binnenlandse Zaken en Koninkrijksrelaties. Het team Data, AI en Algoritmes stelt de informatie samen, op basis van:

- alle wetten en regels, zoals de [AI-verordening](https://eur-lex.europa.eu/legal-content/NL/TXT/?uri=CELEX%3A32024R1689&qid=1723226664277) en de [Algemene wet bestuursrecht](https://wetten.overheid.nl/BWBR0005537/2022-11-01/1)

```

- belangrijke instrumenten, zoals de [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes) (Impact Assessment Mensenrechten en Algoritmes)

> - adviezen en ervaringen van experts, zoals wetenschappers
> ## Verschillende versies

Reviewer: ```suggestion
- adviezen en ervaringen van experts, zoals wetenschappers

## Verschillende versies
```
